src/cart/crt_hg.c

```diff
@@ -824,7 +1051,9 @@ crt_rpc_handler_common(hg_handle_t hg_hdl)
 	bool			 is_coll_req = false;
 	int			 rc = 0;
 	struct crt_rpc_priv	 rpc_tmp = {0};
-
+	d_list_t		*halink;
+	struct crt_ha_mapping 	*ha;
+ 	struct ha_entry 	ha_entry;
```

Comment:
(style)  code indent should use tabs where possible
(style)  please, no space before tabs